corehq/apps/analytics/static/analytix/js/hubspot.js

```diff
@@ -171,50 +171,24 @@ hqDefine('analytix/js/hubspot', [
                 });
             },
             onFormSubmit: function ($form) {
-                $('#get-trial-cta-calendar-content').fadeIn();
-                $('#get-trial-cta-form-content').addClass('hide');
-
                 var email = $form.find('[name="email"]').val(),
                     firstname = $form.find('[name="firstname"]').val(),
                     lastname = $form.find('[name="lastname"]').val(),
                     newUrl = document.location.href + '?email=' + email + '&name=' + firstname + '%20' + lastname;
 
                 kissmetrics.track.event("Get Trial Workflow - Contact Info Received");
-
+                
                 // This nastiness is required for Schedule Once to auto-fill
-                // required fields. Sending snark the way of the S.O. devs...
+                // required fields. Sending snark++ the way of the S.O. devs...
                 window.history.pushState({}, document.title, newUrl);
-
                 // Causes the Schedule Once form to populate the element
```

Comment:
why was this comment not removed if the code it refers to was removed?

Reply:
or maybe we can move the comment if it's still relevant?

Reply:
Yeah. It makes no sense here. 
Sure will remove these from here.